include/boost/geometry/algorithms/detail/buffer/buffer_inserter.hpp

```diff
@@ -921,7 +921,7 @@ template
     typename JoinStrategy,
     typename EndStrategy,
     typename PointStrategy,
-    typename IntersectionStrategy,
+    typename Strategy,
```

Comment:
Because this is the umbrella strategy and the other ones are buffer-only strategies/parameters/selectors. I probably should've called it `Strategies` or `UmbrellaStrategy`.

Reply:
OK, I like `UmbrellaStrategy` naming but it's up to you.